app/src/app/initApp.js

```diff
@@ -91,10 +92,10 @@ export default () => {
       if (data.url) lastUrl = data.url
     }
 
-    if (lastUrl) {
-      loadApp(lastUrl)
-    } else if (env.chooseDomainDisabled) {
+    if (env.chooseDomainDisabled) {
       loadApp()
+    } else if (lastUrl && parseUrl(lastUrl).host === state.host.domain) {
```

Comment:
the check on the domain is to prevent this scenario:

* user has a graperc with a domain
* closes the app
* removes graperc
* opens the app
* it shoudn't go to the login page associated with the old domain